hypervisor/arch/x86/configs/pci_dev.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2019 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <vm_config.h>
+#include <pci_devices.h>
+
+static uint16_t pcidev_config_num = 0U;
+static struct acrn_vm_pci_dev_config pcidev_config[CONFIG_MAX_PCI_DEV_NUM] = {};
+
+/*
+ * @pre pdev != NULL;
+ */
+static bool is_allocated_to_prelaunched_vm(struct pci_pdev *pdev)
+{
+	bool found = false;
+	uint16_t vmid;
+	uint32_t pci_idx;
+	struct acrn_vm_config *vm_config;
+	struct acrn_vm_pci_dev_config *dev_config;
+
+	for (vmid = 0U; vmid < CONFIG_MAX_VM_NUM; vmid ++) {
```

Comment:
for (vmid = 0U; vmid < CONFIG_MAX_VM_NUM && !found; vmid ++) {

??

Reply:
The MISRA-C doesn't encourage do thing like this. Maybe I'm wrong. But I didn't see any use case like this.